### PR TITLE
chore(docs): digest trigger documentation

### PR DESCRIPTION
**Pull request by PHWaechtler**
related to OPT-6340

Add digest crontrigger config and adjust digest documentation accordingly.

### CODE REVIEW

File: docs/components/optimize/userguide/processes.md [hunk 1]
```diff
@@ -20,5 +20,6 @@ Once a report is configured as KPI, its status can be seen on the **Processes Ov
 
 ## Configuring process owner and digests
 
-The process can be configured by clicking the 'Configure' button when hovering over the process. From this modal, you can change the owner of the process, as well as enable/disable the process digest. The process digest is a scheduled email report summarizing the current and previous state of the KPI reports for that process. It will be emailed to the owner of that process at the configured schedule. Please note that process digests are an alpha feature.
+The process can be configured by clicking the 'Configure' button when hovering over the process. From this modal, you can change the owner of the process, as well as enable/disable the process digest. The process digest is a scheduled email report summarizing the current and previous state of the KPI reports for that process. It will be emailed to the owner of that process at globally configurable regular intervals. Please note that process digests are an alpha feature.
 ![Configure Process](./img/configureProcess.png)
+do
```
Comment (RomanJRW) [inline docs/components/optimize/userguide/processes.md:25]: dooooooo

File: docs/components/optimize/userguide/processes.md [hunk 2]
```diff
@@ -20,5 +20,6 @@ Once a report is configured as KPI, its status can be seen on the **Processes Ov
 
 ## Configuring process owner and digests
 
-The process can be configured by clicking the 'Configure' button when hovering over the process. From this modal, you can change the owner of the process, as well as enable/disable the process digest. The process digest is a scheduled email report summarizing the current and previous state of the KPI reports for that process. It will be emailed to the owner of that process at the configured schedule. Please note that process digests are an alpha feature.
+The process can be configured by clicking the 'Configure' button when hovering over the process. From this modal, you can change the owner of the process, as well as enable/disable the process digest. The process digest is a scheduled email report summarizing the current and previous state of the KPI reports for that process. It will be emailed to the owner of that process at globally configurable regular intervals. Please note that process digests are an alpha feature.
```
Comment (RomanJRW) [inline docs/components/optimize/userguide/processes.md:23]: do you think it's worth linking to the config here? Or is this more user focused than technical?

**Review by RomanJRW — APPROVED** [review]
LGTM just a mistake and a tiny suggestions?

File: docs/components/optimize/userguide/processes.md (hunk 2 above)
Comment (PHWaechtler) [reply docs/components/optimize/userguide/processes.md:23]: I wasn't sure if we want to avoid links that need maintained later on, but can't hurt

**Review by christinaausley — APPROVED** [review]
Minor adjustments in latest commit:
- Removal of "please"
- Sentence case headers

LGTM!

**Pull request merged**